src/components/Button.tsx

```diff
@@ -1,62 +1,92 @@
   const isDisabled = !!disabled;
   const ariaProps = { onPress, isDisabled, ...otherProps, ...menuTriggerProps };
   const { label, icon, variant = "primary", size = "sm", buttonRef } = ariaProps;
   const ref = buttonRef || useRef(null);
-  const { buttonProps, isPressed } = useButton(ariaProps, ref as RefObject<HTMLButtonElement>);
+  const { buttonProps, isPressed } = useButton(
+    {
+      ...ariaProps,
+      onPress: typeof onPress === "string" ? noop : onPress,
+      elementType: typeof onPress === "string" ? "a" : "button",
+    },
+    ref as RefObject<HTMLElement>,
+  );
   const { isFocusVisible, focusProps } = useFocusRing(ariaProps);
   const { hoverProps, isHovered } = useHover(ariaProps);
-  const { baseStyles, hoverStyles, disabledStyles, pressedStyles } = useMemo(() => getButtonStyles(variant, size), [
-    variant,
-    size,
-  ]);
+  const { baseStyles, hoverStyles, disabledStyles, pressedStyles } = useMemo(
+    () => getButtonStyles(variant, size),
+    [variant, size],
+  );
   const focusRingStyles = useMemo(() => (variant === "danger" ? Css.bshDanger.$ : Css.bshFocus.$), [variant]);
 
-  const button = (
-    <button
-      ref={ref}
-      {...buttonProps}
-      {...focusProps}
-      {...hoverProps}
-      css={{
-        ...Css.buttonBase.$,
-        ...baseStyles,
-        ...(isHovered && !isPressed ? hoverStyles : {}),
-        ...(isPressed ? pressedStyles : {}),
-        ...(isDisabled ? { ...disabledStyles, ...Css.cursorNotAllowed.$ } : {}),
-        ...(isFocusVisible ? focusRingStyles : {}),
-      }}
-    >
+  const buttonContent = (
+    <>
       {icon && <Icon xss={iconStyles[size]} icon={icon} />}
       {label}
       {endAdornment && <span css={Css.ml1.$}>{endAdornment}</span>}
-    </button>
+    </>
   );
+  const buttonAttrs = {
+    ref: ref as any,
+    ...buttonProps,
+    ...focusProps,
+    ...hoverProps,
+    css: {
+      ...Css.buttonBase.$,
+      ...baseStyles,
+      ...(isHovered && !isPressed ? hoverStyles : {}),
+      ...(isPressed ? pressedStyles : {}),
+      ...(isDisabled ? { ...disabledStyles, ...Css.cursorNotAllowed.$ } : {}),
+      ...(isFocusVisible ? focusRingStyles : {}),
+    },
+  };
 
-  // If we're disabled b/c of a non-boolean ReactNode, show it in a tooltip
-  if (isDisabled && typeof disabled !== "boolean") {
+  const button =
+    typeof onPress === "string" ? (
+      isAbsoluteUrl(onPress) ? (
+        <a {...buttonAttrs} href={onPress} className={navLink} target="_blank" rel="noreferrer noopener">
+          {buttonContent}
+        </a>
+      ) : (
+        <Link {...buttonAttrs} to={onPress} className={navLink}>
+          {buttonContent}
+        </Link>
+      )
+    ) : (
+      <button {...buttonAttrs}>{buttonContent}</button>
+    );
+
+  // If we're disabled b/c of a non-boolean ReactNode, or the caller specified tooltip text, then show it in a tooltip
+  if ((isDisabled && typeof disabled !== "boolean") || tooltip) {
```

Comment:
There are now two ways of including a Tooltip, which I'm not too excited about. Feels more like bloat of the component. Though, continuing to allow the `disabled` property to be both a boolean and the Tooltip content does save an additional property for buttons utilizing `disabled` as the Tooltip content. Not if that "saving an extra property" is worth the additional logic in the component. Happy to hear other thoughts.

Reply:
Yeah, we didn't anticipate "tooltips for non-disabled buttons" when we first added this...I'm good with the current "there are technically two ways" approach.